src/poker.js

```diff
@@ -1,5 +1,63 @@
-class Poker {
-  // Write your implementation here
+const scoreBoard = {
+  1: 1,
```

Comment:
never seen a 1 card ;-)

Reply:
Just noticed this now as well ahah -.-' don't tell me how to run my casino sir.